cap-ci/pipeline.yaml.tmpl

```diff
@@ -68,6 +68,15 @@ resources:
     regexp: {{ $config.s3.regexp }}
     region_name: {{ $config.s3.region }}
 
+{{- if $config.schedule }}
+- name: schedule-interval
+  type: time
+  source:
+    start: {{ $config.schedule.start }}
```

Comment:
why not just use 

``` 
 type: time
  source:
    interval: 24h
``` 
?

Reply:
may we because we want it to be on specific time.

Reply:
thanks for pointing I missed the `interval` part. Rest is to ensure we trigger during night time hence `nightly`
https://github.com/concourse/time-resource#trigger-on-an-interval-within-time-range